http/org_service.go

```diff
@@ -73,6 +73,24 @@ const (
 	organizationsIDLabelsIDPath      = "/api/v2/orgs/:id/labels/:lid"
 )
 
+func (h *OrgHandler) handleGetOrgCheck(w http.ResponseWriter, r *http.Request) interface{} {
```

Comment:
love the helper function!

Reply:
Could this just return a organization pointer?